core/src/main/java/com/homeaway/streamplatform/streamregistry/dto/JsonToAvroDTO.java

```diff
@@ -97,6 +95,36 @@ public static AvroStream convertJsonToAvro(Stream jsonStream, OperationType oper
 
     }
 
```

Comment:
I'm not a fan of static methods... unless they are absolutely necessary (which is rare).
This method seems to be cohesive with whatever dao ends up serving source / sink info.